automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/WebBrowserSpec.scala

```diff
@@ -44,7 +46,14 @@ trait WebBrowserSpec extends WebBrowserUtil with ExceptionHandling with LazyLogg
   private def runLocalChrome(testCode: (WebDriver) => Any) = {
     val service = new ChromeDriverService.Builder().usingDriverExecutable(new File(Config.ChromeSettings.chromDriverPath)).usingAnyFreePort().build()
     service.start()
-    implicit val driver = new RemoteWebDriver(service.getUrl, DesiredCapabilities.chrome())
+
+    // run in incognito
```

Comment:
you could pull this out into a function instead of having the same code twice:
```  def makeCapabilitesIncognito: DesiredCapabilities = { val options = new ChromeOptions
  options.addArguments("--incognito")
  val capabilities = DesiredCapabilities.chrome
  capabilities.setCapability(ChromeOptions.CAPABILITY, options)
}```